cmd/openfga/main_test.go

```diff
@@ -817,6 +819,275 @@ func TestCheckWorkflows(t *testing.T) {
 	})
 }
 
+// TestExpandWorkflows are tests that involve workflows that define assertions for
+// Expands against multi-model stores etc..
+func TestExpandWorkflows(t *testing.T) {
+
+	tester, err := newOpenFGATester(t)
+	require.NoError(t, err)
+	defer tester.Cleanup()
+
+	conn := connect(t, tester)
+	defer conn.Close()
+
+	client := openfgapb.NewOpenFGAServiceClient(conn)
+
+	/*
+	 * TypedWildcardsFromOtherModelsIgnored ensures that a typed wildcard introduced
+	 * from a prior model does not impact the Expand outcome of a model that should not
+	 * involve it. For example,
+	 *
+	 * type user
+	 * type document
+	 *   relations
+	 *     define viewer: [user, user:*]
+	 *
+	 * write(document:1#viewer@user:*)
+	 * write(document:1#viewer@user:jon)
+	 * Expand(document:1#viewer) --> {tree: {root: {name: document:1#viewer, leaf: {users: [user:*, user:jon]}}}}
+	 *
+	 * type user
+	 * type document
+	 *	relations
+	 *	  define viewer: [user]
+	 *
+	 * Expand(document:1#viewer) --> {tree: {root: {name: document:1#viewer, leaf: {users: [user:jon]}}}}
+	 *
+	 * type employee
+	 * type document
+	 *   relations
+	 *     define viewer: [employee]
+	 *
+	 * type user
+	 * type employee
+	 * type document
+	 *   relations
+	 *     define viewer: [employee]
+	 * Expand(document:1#viewer) --> {tree: {root: {name: document:1#viewer, leaf: {users: []}}}}
```

Comment:
Just to be clear, at this point,

`Expand(document:1#viewer, latestModelId)` would return `{tree: {root: {name: document:1#viewer, leaf: {users: []}}}}`

but 

`Expand(document:1#viewer, firstModelId)` would still return `{tree: {root: {name: document:1#viewer, leaf: {users: [user:*, user:jon]}}}}`

right?

Reply:
Correct 👍